Dockerfile.ocp

```diff
@@ -1,4 +1,4 @@
-FROM openshift/origin-base AS builder
+FROM ironic-builder AS builder
```

Comment:
So is "ironic-builder" essentially a alias for ubi ?

Reply:
@derekhiggins the "ironic-builder" image is an alias for another image that it's built inside OCP specifically in the ironic image configuration, there is not a generic ubi8 based image that is directly accessible at the moment, so I had to sneak one in :)
see https://github.com/openshift/release/pull/7010 and https://github.com/openshift/release/blob/master/ci-operator/config/openshift/ironic-image/openshift-ironic-image-master.yaml
I'm planning to reuse the same process for other images that need a builder, an example could be the ipa-downloader if we decide to build our own ipa ramdisk.